src/state/ducks/index.ts

```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from "redux";
+import studentReducer from "./student/reducer";
+import { all, fork } from "redux-saga/effects";
+import studentSaga from "./student/saga";
+
+export const rootReducer = combineReducers({
+  studentReducer,
```

Comment:
```suggestion
  student: studentReducer,
```
if You do like this student key will be created in reducer. you current code is creating a studentReducer key inside redux.